pyproject.toml

```diff
@@ -52,40 +67,129 @@ selenium = ">=4.1.0"
 importlib_metadata = {version = "*", python = "3.8.*"}
 
 # convenience packages for development of screenpy only
+black = {version = "*", optional = true}
+coverage = {version = "*", optional = true}
+isort = {version = "*", optional = true}
```

Comment:
Do we still need `isort` anymore? Doesn't `ruff` do the needful?

Reply:
Technically no. 
I've still been using it until ruff implements the combine imports to single line feature.  But otherwise no reason to have it in there still.

Reply:
All right, i don't really see a problem with it for now. 👍